apitests/spec/device_rules_spec.ts

```diff
@@ -46,23 +52,24 @@ describe('device rules api', () => {
                 expect(result.data).toBeTruthy();
                 expect(result.data.length).toBeGreaterThan(0);
                 expect(result.data[0]).toBeTruthy();
- +              expect(result.data[0].ruleId).toBeTruthy();
- +              expect(result.data[0].enabledState).toBeDefined();
+                expect(result.data[0].ruleId).toBeTruthy();
+                expect(result.data[0].enabledState).toBeDefined();
                 expect(result.draw).toBeDefined();
- +              expect(result.recordsTotal).toBeDefined();
- +              expect(result.recordsFiltered).toBeDefined();
+                expect(result.recordsTotal).toBeDefined();
+                expect(result.recordsFiltered).toBeDefined();
                 done();
             });
         });
     });
 
-    describe('create new device rule', () => {
-          it('should create new rule', (done) => {
+    xdescribe('create new device rule', () => {
+        it('should create new rule', (done) => {
+            var ruleId:string = "testRule" + Math.random().toString(36).substr(2, 5);
```

Comment:
usually generating guids are better to create randomness for tests. Here's a library you can use to generate them: https://github.com/broofa/node-uuid